VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs

```diff
@@ -50,15 +50,6 @@ public void ThenTheFirstParticipantStatusIsDisplayedAsNotSignedIn(string name, s
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeParticipantPanel.ParticipantStatus(participant.Id)).Text.ToUpper().Trim().Should().Be(status.ToUpper());
         }
 
-        [When(@"the judge dismisses the change camera popup")]
-        [Then(@"the judge dismisses the change camera popup")]
-        public void TheJudgeDismissesTheChangeCameraPopup()
-        {
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeWaitingRoomPage.ChangeDeviceButton).Displayed.Should().BeTrue();
-            _browsers[_c.CurrentUser].Click(JudgeWaitingRoomPage.ChangeDeviceButton);
-            _browsers[_c.CurrentUser].Driver.WaitUntilElementNotVisible(JudgeWaitingRoomPage.ChangeDeviceButton);
-        }
-
```

Comment:
Are you removing this completely? Is there no option to change the camera anymore?

Reply:
It's does not show on first entrance anymore, but should be display popup on press button.